src/styles/index.scss

```
@@ -1,3 +1,204 @@
+@import 'mixins';
+@import 'main';
+@import 'stars';
+
+$theme-color: #00acdc;
+$text-color: #060b35;
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+
+  &__link {
+    margin-left: 50px;
+
+    @include flex-center;
+  }
+}
+
+.navigation {
+  height: 100%;
+  margin-right: 50px;
+  align-items: stretch;
+
+  &__tabs {
+    height: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: stretch;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__tab {
+    display: flex;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  &__tabs &__tab:first-child {
+    margin-left: 0;
+  }
+
+  &__tabs &__tab:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    margin: 0;
+    padding: 0;
+    font-size: 12px;
+    font-weight: 500;
+    color: $text-color;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    @include flex-center;
+
+    &:hover {
+      color: $theme-color;
+      cursor: pointer;
+    }
+  }
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url(../images/imac.jpeg) center 32px/160px 134px no-repeat;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__label {
+    margin-top: 206px;
+    width: 166px;
```

Fix it everywhere and add paddings for the card containter
```suggestion
    width: 100%;
```